lib/module/doing.go

```diff
@@ -0,0 +1,33 @@
+package module
+
+import (
+	"fmt"
+	"github.com/davidscholberg/irkbot/lib/configure"
+	"github.com/davidscholberg/irkbot/lib/message"
+	"math/rand"
+	"strings"
+	"time"
+)
+
+func HelpDoing() []string {
+	s := "doing [subject] - sun is not doing, [subject] is doing; defaults to linux"
+	return []string{s}
+}
+
+func Doing(cfg *configure.Config, in *message.InboundMsg, actions *Actions) {
+	rand.Seed(time.Now().Unix())
+	doingNot := make([]string, 0)
+	doingNot = append(doingNot,
+		"sun is",
+		"stars are",
+		"trees are",
+		"moon is",
+		"planets are",
+		"galaxies are")
```

Comment:
This should really be an array literal instead of a slice since the elements are fixed at compile time.